src/cli/info.rs

```diff
@@ -68,22 +93,114 @@ impl Display for Info {
+                writeln!(f, "{:20}: {}", "Last updated", update_time)?;
+            }
+
+            if !pi.platforms.is_empty() {
+                for (i, p) in pi.platforms.iter().enumerate() {
+                    if i == 0 {
+                        writeln!(f, "{:20}: {}", "Target platforms", p)?;
+                    } else {
+                        writeln!(f, "{:20}: {}", "", p)?;
+                    }
+                }
+            }
+
+            if !pi.tasks.is_empty() {
+                for (i, t) in pi.tasks.iter().enumerate() {
+                    if i == 0 {
+                        writeln!(f, "{:20}: {}", "Tasks", t)?;
+                    } else {
+                        writeln!(f, "{:20}: {}", "", t)?;
+                    }
+                }
             }
         }
 
         Ok(())
     }
 }
 
+/// Returns the size of a directory
+fn dir_size(path: impl Into<PathBuf>) -> miette::Result<String> {
+    fn dir_size(mut dir: fs::ReadDir) -> miette::Result<u64> {
+        dir.try_fold(0, |acc, file| {
+            let file = file.into_diagnostic()?;
+            let size = match file.metadata().into_diagnostic()? {
+                data if data.is_dir() => dir_size(fs::read_dir(file.path()).into_diagnostic()?)?,
+                data => data.len(),
+            };
+            Ok(acc + size)
+        })
+    }
+
+    let size = dir_size(fs::read_dir(path.into()).into_diagnostic()?)?;
+    Ok(format!("{} MiB", size / 1024 / 1024))
+}
+
+/// Returns last update time of file, formatted: DD-MM-YYYY H:M:S
+fn last_updated(path: impl Into<PathBuf>) -> miette::Result<String> {
+    let time = fs::metadata(path.into())
+        .into_diagnostic()?
+        .modified()
+        .into_diagnostic()?;
+    let formated_time = DateTime::<Local>::from(time)
+        .format("%d-%m-%Y %H:%M:%S")
+        .to_string();
+
+    Ok(formated_time)
+}
+
+/// Returns number of dependencies on current platform
+fn dependency_count(project: &Project) -> miette::Result<u64> {
+    let dep_count = project
+        .all_dependencies(Platform::current())?
+        .keys()
+        .cloned()
+        .fold(0, |acc, _| acc + 1);
+
+    Ok(dep_count)
+}
+
 pub async fn execute(args: Args) -> miette::Result<()> {
     let project = Project::load_or_else_discover(args.manifest_path.as_deref()).ok();
 
+    let project_clone = project.clone().unwrap();
+    let env_path = project_clone.root().join(".pixi");
+    let cache_dir = rattler::default_cache_dir().ok();
+    let cache_dir_clone = cache_dir.clone();
+
+    let (environment_size, cache_size) = if args.extended {
+        await_in_progress(
+            "fetching cache",
+            spawn_blocking(|| {
+                let env_size = dir_size(env_path).ok();
+                let cache_size = dir_size(cache_dir_clone.unwrap()).ok();
+                (env_size, cache_size)
+            }),
+        )
+        .await
+        .into_diagnostic()?
```

Comment:
This panicks when run outside of a project. 

I tried it locally and think this is a better way of getting the data there.  By just moving the env_dir and the cache dir instead of the project. 

```rust
    let cache_dir = rattler::default_cache_dir().map_err(|_| miette::miette!("could not determine default cache directory"))?;
    let (environment_size, cache_size) = if args.extended {
        let cache_dir = cache_dir.clone();
        let env_dir = project.as_ref().map(|p | p.root().join(".pixi"));
        await_in_progress(
            "fetching cache",
            spawn_blocking(move || {
                let env_size = env_dir.and_then(|e| dir_size(e).ok());
                let cache_size = dir_size(&cache_dir).ok();
                (env_size, cache_size)
            }),
        )
        .await
        .into_diagnostic()?
```

Reply:
Sure, I'll make this change. This is way better.